pages/utils/email.py

```diff
@@ -139,7 +144,12 @@ def emails(self):
     def compose(self):
         """Return a composition page."""
         self.find_element(*self._compose_locator).click()
-        return Compose(self.selenium)
+        return Compose(self.driver)
+
+    @property
+    def openedmail(self):
```

Comment:
python names should be words separated with an underscore:
openedmail => opened_mail